### PR TITLE
ticket 4

### PR DESCRIPTION
**Pull request by Masonward99**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -35,9 +35,41 @@ describe('GET/api/categories', () => {
...
+            .then((result) => {
+                expect(result.body.review).toHaveProperty('review_id')
+                expect(result.body.review).toHaveProperty("title")
+                expect(result.body.review).toHaveProperty("review_body")
+                expect(result.body.review).toHaveProperty("designer");
+                expect(result.body.review).toHaveProperty("review_img_url");
+                expect(result.body.review).toHaveProperty("votes");
+                expect(result.body.review).toHaveProperty("category");
+                expect(result.body.review).toHaveProperty("owner");
+                expect(result.body.review).toHaveProperty("created_at");
```
Comment (B7Bprog) [inline __tests__/app.test.js:55]: Please also check the actual ID, to make sure we've got the right review back.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -35,9 +35,41 @@ describe('GET/api/categories', () => {
...
+            })
+        .then()
+          
+    })
+    it('returns 404 when there are no reviews with that id', () => {
+        return request(app)
+            .get('/api/reviews/44')
+            .expect(404)
+            .then()
```
Comment (B7Bprog) [inline __tests__/app.test.js:64]: Please expect a message here as well. Something like 'not found'.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -35,9 +35,41 @@ describe('GET/api/categories', () => {
...
+    })
+    it('returns 400 when give a bad input', () => {
+        return request(app)
+            .get('/api/reviews/cat')
+            .expect(400)
+            .then()
```
Comment (B7Bprog) [inline __tests__/app.test.js:70]: A message must be expected here too, for example: 'not-an-id'.

File: db/app.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const { getCategories } = require('../controllers/categories.controllers');
 const { getEndpoints } = require('../controllers/api.controllers');
+const { getReviewById } = require('../controllers/reviews.controllers');
 const app = express();
 
 app.use(express.json())
```
Comment (B7Bprog) [inline db/app.js:7]: This should not be included yet, as it's not part of the task.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Mason, please check my comments and refactor your code, then send us another PR with it. 👍

**Review by Rayhaan2k9 — APPROVED** [review]
Nice work, Merge away! 😄

**Pull request merged**